lib/web_ui/dev/test_runner.dart

```diff
@@ -26,7 +26,7 @@ class TestCommand extends Command<bool> with ArgUtils<bool> {
   TestCommand() {
     argParser
       ..addFlag(
-        'debug',
+        'debug-browser',
```

Comment:
Yeah, that is probably a better name. One detail is that this option also causes us to run in a window instead of headless, but that probably is fine to just leave implied.

Reply:
Yep, that's how it works in the flutter tool too.